markdown/api-docs/cart-and-checkout/embedded-checkout.md

```diff
@@ -0,0 +1,335 @@
+
+## 2: Create a Channel
+
+In order to allow your external website to serve the BigCommerce checkout, you must first create a new Channel using the Channels API.
+
+To do this, make a POST request to `https://api.bigcommerce.com/stores/{{store_hash}}/V3/channels`
+
+<!--
+title: "POST to Channels"
+subtitle: ""
+lineNumbers: true
+-->
+
+**Create Channel POST**
+
+```js
+{
+    "type": "storefront",
+    "platform": "custom",
+    "name": "https://bc-react.herokuapp.com"
+}
+
+```
+
+The response will be an `id` which you will use as the `channel_id` in future requests. Let's assume it's `15001` for this guide.
+
+Once you create a Channel, it appears in your BigCommerce Control Panel under Products > Listed On. Here products can be chosen to be listen on each desired channel. The Orders section will now also include a filter for your channel.
+
+<a href='#embedded-checkout_create-site' aria-hidden='true' class='block-anchor'  id='embedded-checkout_create-site'><i aria-hidden='true' class='linkify icon'></i></a>
+
+## 3: Create a Site
+
+Next you will need to create a site for the channel. Make a POST to `https://api.bigcommerce.com/stores/{{store_hash}}/v3/channels/{{channel_id}}/site`
+
+<!--
+title: "POST to Channels"
+subtitle: ""
+lineNumbers: true
+-->
+
+**Create Site POST**
+
+```js
+{
+    "channel_id": 15001,
+    "url": "https://bc-react.herokuapp.com"
+}
+
+```
+
+This returns `id` which you will use as the `site_id` in future requests. The `url` value is the base path for all other routes you define for the site.
+
+
+<a href='embedded-checkout_create-cart' aria-hidden='true' class='block-anchor'  id='embedded-checkout_create-cart'><i aria-hidden='true' class='linkify icon'></i></a>
+
+## 4: Create a Cart
+
+In order to proceed to checkout, an active cart is required. You'll use the [Server-to-Server Cart API](https://developer.bigcommerce.com/api-reference/cart-checkout/server-server-cart-api) to generate it.
+
+We'll also use the [Catalog API](https://developer.bigcommerce.com/api-reference/catalog/catalog-api) to obtain `product_id`s and/or `variant_id`s for the items we need to build a cart. See [Catalog API](https://developer.bigcommerce.com/api-reference/catalog/catalog-api/products/getproductbyid) for more information on how to GET a product.
+
+Make the POST with your product information to the Carts endpoint: `https://api.bigcommerce.com/stores/{{store_hash}}/v3/carts`
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--">
+<div class="HubBlock-content">
+    
+<!-- theme:  -->
+
+For products without options, you only need to send `quantity` and `product_id` in a `line_items` object to create a cart.
+
+</div>
+</div>
+</div>
+
+<!--
+title: "Create Cart POST"
+subtitle: ""
+lineNumbers: true
+-->
+
+**Create Cart POST**
+
+```js
+{
+  "channel_id": 15001,
+  "line_items": [
+    {
+      "quantity": 1,
+      "product_id": 80,
+      "variant_id": 64
+    }
+  ]
+}
+
+```
```

Comment:
This section could be shortened to: 

```markdown
An active cart is required to proceed to checkout. To create one, send a `POST` request to the [Server-to-Server Cart API's](https://developer.bigcommerce.com/api-reference/cart-checkout/server-server-cart-api)  `/cart` endpoint: 

> **`POST`**  `https://api.bigcommerce.com/stores/{{store_hash}}/v3/carts`

\```json
{
  "channel_id": 15001,
  "line_items": [
    {
      "quantity": 1,
      "product_id": 80,
      "variant_id": 64
    }
  ]
}
\```

Contained in the response is a `UUID` -- this UUID is the `cart_id` used in subsequent requests. 

div class="HubBlock--callout">
<div class="CalloutBlock--info">
<div class="HubBlock-content">

 <!-- theme: info -->

For products without options, you only need to send `quantity` and `product_id` in a `line_items` object to create a cart.

</div>
</div>
</div>

```